Melee Damage Standardization and Improvements

LemonBlemon opened the pull request:
Melee weapon damage makes no sense at the moment, this PR is a good first step to making them more reasonable

<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
Many melee weapons have been tweaked to be more useful based on rarity, size, and purpose. Generally large-sized weapons like sabres, katanas, and Cutlass' deal 32 slash (the damage the machete already has), and huge-sized weapons should deal around 40. Weapons that are supposed to deal more structural damage have also had that increased somewhat, mostly due to the existence of the breaching hammer.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
I made this because i was tired of seeing 15 slash on the plasteel arming sword (which is the same damage as a cybersun pen btw), and the 22 damage on the katana (making it effectively a larger, slower, combat knife). This shouldn't really effect the main faction pvp balance all that much, it should just shorten the gap in power between melee weapons and guns.

## How to test
<!-- Describe the way it can be tested -->
Spawn the altered weapons, hit some urists.

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [ x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ x] I have added media to this PR or it does not require an ingame showcase.
- [x ] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- tweak: Increased several melee weapon's damage values to make them more consistent with Mono's other changes to melee combat.
- tweak: Increased several melee weapon's structural damage values to make them actually work as structural weapons.
- tweak: Eldritch Blade should make noises now
-->

LemonBlemon committed:
Melee Damage Standardization and Improvements

Many melee weapons have been tweaked to be more useful based on rarity, size, and purpose. Generally large-sized weapons like sabres, katanas, and Cutlass' deal 32 slash (the damage the machete already has), and huge-sized weapons should deal around 40. Weapons that are supposed to deal more structural damage have also had that increased somewhat, mostly due to the existence of the breaching hammer.
I made this because i was tired of seeing 15 slash on the plasteel arming sword (which is the same damage as a cybersun pen btw), and the 22 damage on the katana (making it effectively a larger, slower, combat knife).

monolith8319 committed:
chore: automatically update REUSE headers

OneZerooo0 commented on Resources/Prototypes/Entities/Objects/Weapons/Melee/hammers.yml:
```diff
@@ -19,8 +25,8 @@
-        Blunt: 10
-        Structural: 10
+        Blunt: 15 # increased damage from 10 to 15 so its no longer worse than a combat knife
+        Structural: 10 # increased damage from 10 to 30 so its actually usable as a structural weapon
```
didn't actually change the struct damage value

OneZerooo0 reviewed (changes requested):
add Mono: to all of your comments

LemonBlemon committed:
Update chainsaw.yml

fixed comments on chainsaw.yml

LemonBlemon committed:
Update cult.yml

fixed the changelogs on cult.yml, also fixed the eldritch blade being completely silent

LemonBlemon committed:
Update e_sword.yml

Fixed comments on e_sword.yml

LemonBlemon committed:
Update hammers.yml

Fixed comments on hammers.yml, also oops i forgot to actually change the structural damage

LemonBlemon committed:
Update sword.yml

Fixed comments on sword.yml

LemonBlemon committed:
Update sword.yml

Fixed comments on the Goob sword.yml file

LemonBlemon committed:
Update sword.yml

Fixed comments on Monolith sword.yml file

LemonBlemon committed:
Update sword.yml

fixed comments on NF sword.yml

LemonBlemon committed:
Update cult.yml

Re-fixed the audio to hopefully work this time

LemonBlemon commented:
I hope i fixed it? this is my first PR so please abide my lack of skill and knowledge

Danchi299 commented on Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml:
```diff
@@ -166,7 +169,7 @@
     damage:
       types:
-        Slash: 32
+        Slash: 32 # Mono this amount of slash is now treated as the "base" for all large sized melee weapons
```
Isn't it also the damage of Combat Knife / Kukri?

LemonBlemon replied:
combat knife is 22, which is what the katana used to also do before this pr

Danchi299 commented:
I think Throwing Knife has same thrown damage as Combat Knife.

LemonBlemon committed:
Merge branch 'main' into suffering

monolith8319 committed:
chore: automatically update REUSE headers

LemonBlemon commented:
there i actually fixed it now
